### DISCUSSION TITLE
How to install okd 3.11 in 2023?

### DISCUSSION THREAD

**Discussion by m0n0l1br3** [okd-project/okd#1770] [General]
Hello!

OKD 3.11 reached its end of lifetime on june 2022.

It so happens that my project highly relies on okd 3.11 and upgrading to 4+ would result in rewriting a lot of code.

Is there any way to install okd 3.11 in 2023? The existing playbook doesnt work, sources of 3.11 are deleted :-(

**Comment by LorbusChris** [comment]
Unfortunately I don't see a way to do that. This tracker is for OKD 4, not Origin 3, so closing.
